lib/web_ui/test/canvaskit/common.dart

```diff
@@ -45,10 +46,12 @@ CkPicture paintPicture(
   return recorder.endRecording();
 }
 
-Future<void> matchSceneGolden(String goldenFile, LayerScene scene, {
+Future<void> matchSceneGolden(
+  String goldenFile,
+  LayerScene scene, {
   required ui.Rect region,
 }) async {
-  CanvasKitRenderer.instance.rasterizer.draw(scene.layerTree);
+  CanvasKitRenderer.instance.renderScene(scene, implicitView);
   await matchGoldenFile(goldenFile, region: region);
```

Comment:
Maybe @goderbauer can provide more detail, but apparently we render scenes for golden tests at random points, which is making this difficult:

* https://github.com/flutter/flutter/issues/137073

(Not sure how hard would it be to ensure the tests are in a state where `render` *can* be performed!)

Reply:
Added a TODO to track this. We should follow the render rule to unblock other platforms from enforcing it.